features/items.md

```diff
@@ -0,0 +1,237 @@
+# Items
+
+## Introduction
+OpenHAB has a strict separation between the physical world (the “things”, see below) and the application, which is built around the notion of “items” (also called the virtual layer).
+
+Items represent functionality that is used by the application (mainly user interfaces or automation logic). Items have a state and are used through events. They are objects that can be read from, or written to, in order to interact with them.
```

Comment:
That's not how it is, having one linebreak between sentences will not change the presentation. Only after two line breaks, a paragraph break will be rendered by markdown.
So:

* No effect on the presentation in html / on the end user
* Better maintainability by the help of git.